### ISSUE TITLE
[feature] Add support for vint's security

### ISSUE THREAD

**Issue by Freed-Wu** [iamcco/diagnostic-languageserver#70]
By default:

```json
"vint": {
  "command": "vint",
  "debounce": 100,
  "args": [ "--enable-neovim", "-"],
  "offsetLine": 0,
  "offsetColumn": 0,
  "sourceName": "vint",
  "formatLines": 1,
  "formatPattern": [
    "[^:]+:(\\d+):(\\d+):\\s*(.*)(\\r|\\n)*$",
    {
      "line": 1,
      "column": 2,
      "message": 3
    }
  ]
}
```

There exists a problem, the security of vint message cannot be known. coc will consider all the message to error. It is incorrect.

A solution is change the format to contain [sevrity](https://github.com/Vimjas/vint/wiki/Vint-linting-policy-summary)

```json
      "vint": {
        "command": "vint",
        "debounce": 100,
        "args": ["-f", "{file_path}:{line_number}:{column_number}: {severity}! {description}", "-"],
        "offsetLine": 0,
        "offsetColumn": 0,
        "sourceName": "vint",
        "formatLines": 1,
        "formatPattern": [
          "[^:]+:(\\d+):(\\d+):\\s*([^!]*)! (.*)(\\r|\\n)*$",
          {
            "line": 1,
            "column": 2,
            "security": 3,
            "message": 4
          }
        ],
        "securities": {
          "error": "error",
          "warning": "warning",
          "style_problem": "info"
        }
      }
```

Now the security is warning not error:

![image](https://user-images.githubusercontent.com/32936898/134008061-0ee61d9b-f6d6-4b6d-9c8b-d608f8c0d5ba.png)

Thanks!

**Comment by iamcco** [comment]
pr is welcome.
